src/test/java/com/github/jmchilton/blend4j/galaxy/HistoriesTest.java

```diff
@@ -427,18 +427,6 @@ public void testDownloadDatasetSuccess() throws Exception {
   }
   
   /**
-   * Tests out downloading a dataset with an invalid history id.
-   * @throws IOException 
-   */
-  @Test(expectedExceptions=RuntimeException.class)
-  public void testDownloadDatasetFailDatasetId() throws IOException {    
```

Comment:
This is due to an API change in newer versions of Galaxy.  It used to be that getting a dataset with an invalid history id would fail, but now it will still give you the proper dataset (the dataset id is unique across all histories).

This change is also why we have an `@Ignore` change in our IRIDA code for one test, first introduced in http://irida.corefacility.ca/irida/irida/merge_requests/244 when we upgraded Galaxy.